### ISSUE TITLE
Option to DL resized images.

### ISSUE THREAD

**Issue by junh1024** [CryShana/Sankaku-Channel-Downloader#19]
Can you add an Option to Download resized images?

**Comment by CryShana** [comment]
Yeah. I think I'm already getting the resized image url. I just need to add a checkbox and adjust some logic. Will add it as soon as I have the time. Today or tomorrow.
